view/frontend/templates/product/view/attachments.phtml

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Attachments
+ */
+?>
+
+<?php $ajaxurl = $block->getFormAction(); ?>
+<?php $collection = $block->getCollection(); ?>
+
+<?php if ($collection && $collection->count()>0) { ?>
+
+<p> Our automated regulatory documentation process offers quick access to information, 
```

Comment:
@NandhiniKarthi Always use translate string methods for static text to enable multi-language